skip-tests.json

```diff
@@ -44,9 +44,8 @@
         }
     },
     "binance": {
-        "httpProxy": "http://51.83.140.52:11230",
+        "proxyHttps": "http://51.83.140.52:11230",
```

Comment:
@ttodua I missed this rename but I think we should still support it, actually I think it sounds slightly better "httsProxy/socksProxy/etc" than "proxyHtts/proxySocks"

Reply:
about names, in this case we have quite many `proxy` props, so, scattering them into different "start-end" would be more confusing, as use starting with `proxy...` will get all references displayed and this might be rare case when it's better this way (if we had only one prop, then we could easily reverse), i think so in this case for proxies.
but as you'll decide...

Reply:
@ttodua I understand your point, but for instance

node fetch
` const proxyAgent = new HttpsProxyAgent('http://46.250.171.31:8080');` 

AIOHTTP/requests
` But you can do so by passing trust_env=True into aiohttp.ClientSession constructor for extracting proxy configuration from HTTP_PROXY, HTTPS_PROXY, WS_PROXY or WSS_PROXY `

etc

They all seem to follow the other approach, protocol first and proxy second, so that format `httsProxy` etc seems a bit more standard to me